.github/workflows/test_and_deploy.yml

```diff
@@ -27,7 +29,9 @@ jobs:
         run: npm ci
 
       - name: Build and package library
-        run: npm run package
+        run: |
+          npm run lint:commit
```

Comment:
I guess having this in the run package block is weird. Maybe just add it to `test` script of the package.json and we later clean the scripts up.